tests/go-validation/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -x
+set -eu
+
+DEBUG="${DEBUG:-false}"
+
+TRIM_JOB_NAME=$(basename "$JOB_NAME")
+
+if [ "false" != "${DEBUG}" ]; then
+    echo "Environment:"
+    env | sort
+fi
+
+count=0
+while [[ 3 -gt $count ]]; do
+    docker build . -f tests/go-validation/Dockerfile.v1api -t rancher-validation-"${TRIM_JOB_NAME}""${BUILD_NUMBER}"  
+
+    if [[ $? -eq 0 ]]; then break; fi
+    count=$(($count + 1))
```

Comment:
Nitpick: no need for '$' on count, see https://github.com/koalaman/shellcheck/wiki/SC2004 for more information

Reply:
Fixed